1 tables.sql

```diff
@@ -0,0 +1,8 @@
+create database TrackGermDB

```

Comment:
I was unable to run the file more than once. I think you may be missing a `drop db statement`.

Reply:
added commit to fix these problems